app/models/info_request.rb

```diff
@@ -146,6 +146,8 @@ def self.admin_title
   scope :is_searchable, Prominence::SearchableQuery.new
   scope :embargoed, Prominence::EmbargoedQuery.new
   scope :not_embargoed, Prominence::NotEmbargoedQuery.new
+  scope :was_embargoed, Prominence::WasEmbargoedQuery.new
```

Comment:
I think I'd call this `ever_embargoed` since it also includes currently embargoed. "was" implies "did have one but now hasn't" so could be a bit of a gotcha.